### PR TITLE
Change miniver fallback version from `unknown` to `0.0.0`.

### PR DESCRIPTION
**Pull request by arangatang**
*Issue #, if available:*

*Description of changes:*
Since #1434 some builds of gluonts dockerfiles fails to build. This happens when the proper release can not be detected properly. Below is an example error:

```
...
Building wheels for collected packages: gluonts, pymeeus
Building wheel for gluonts (PEP 517): started
Building wheel for gluonts (PEP 517): finished with status 'done'
Created wheel for gluonts: filename=gluonts-unknown_gca037a1.dirty-py3-none-any.whl size=2060228 sha256=0215c4a58866852089ad2ee75e950305ebe7d92ee27dae6d8f0666ee928e6287
 Stored in directory: /tmp/pip-ephem-wheel-cache-nw01nvp7/wheels/84/fe/8a/0bf22561f0dcaed3788aee2ee6e95ad1e220e35a278d83522b
WARNING: Built wheel for gluonts is invalid: Metadata 1.2 mandates PEP 440 version, but 'unknown-gca037a1.dirty' is not
...
Successfully built pymeeus
Failed to build gluonts
ERROR: Could not build wheels for gluonts which use PEP 517 and cannot be installed directly
```

The culprit seem to be the `__version__` variable in [_version.py](src/gluonts/_version.py) which in this case is evaluated to `unknown-gca037a1.dirty`.
Replacing the unknown with a default version value such as `0.0.0` resolves the issue. 

```
...
Building wheels for collected packages: gluonts, pymeeus
Building wheel for gluonts (PEP 517): started
Building wheel for gluonts (PEP 517): finished with status 'done'
Created wheel for gluonts: filename=gluonts-0.0.0+g00254ed.dirty-py3-none-any.whl size=2060198 sha256=baa1ca787a42898fe4899afa8ff7988ab9feb4463a6c8665c458382b8f28454b
Stored in directory: /tmp/pip-ephem-wheel-cache-9q8ttprs/wheels/84/fe/8a/0bf22561f0dcaed3788aee2ee6e95ad1e220e35a278d83522b
...
Successfully built gluonts pymeeus
```

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

**Please tag this pr with at least one of these labels to make our release process faster:** BREAKING, new feature, bug fix, other change, dev setup

### CODE REVIEW

**Comment by arangatang** [comment]
Seems that setting default values for a named tuple was not possible prior to python 3.7 and this is why the tests are failing.

File: src/gluonts/_version.py [hunk 1]
```diff
@@ -163,9 +164,9 @@ def get_version_from_git_archive(version_info):
     version_tags = set(r[len(VTAG) :] for r in refs if r.startswith(VTAG))
     if version_tags:
         release, *_ = sorted(version_tags)  # prefer e.g. "2.0" over "2.0rc1"
-        return Version(release, dev=None, labels=None)
+        return Version(release, None, None)
```
Comment (jaheba) [inline src/gluonts/_version.py:167]: Can we revert this to the original version?

File: src/gluonts/_version.py [hunk 2]
```diff
@@ -35,7 +36,7 @@ def get_version(version_file=STATIC_VERSION_FILE):
         if not version:
             version = get_version_from_git_archive(version_info)
         if not version:
-            version = Version("unknown", None, None)
+            version = Version(DEFAULT_RELEASE, 0, 0)
```
Comment (jaheba) [inline src/gluonts/_version.py:39]: Why not 

```suggestion
            version = Version(DEFAULT_RELEASE, None, None)
```

File: src/gluonts/_version.py [hunk 3]
```diff
@@ -35,7 +36,7 @@ def get_version(version_file=STATIC_VERSION_FILE):
         if not version:
             version = get_version_from_git_archive(version_info)
         if not version:
-            version = Version("unknown", None, None)
+            version = Version(DEFAULT_RELEASE, dev=None, labels=None)
```
Comment (jaheba) [inline src/gluonts/_version.py:39]:
```suggestion
            version = Version(DEFAULT_RELEASE, None, None)
```

File: src/gluonts/_version.py [hunk 4]
```diff
@@ -25,6 +25,7 @@
     _package_root_inside_src = False
 
 STATIC_VERSION_FILE = "_static_version.py"
+DEFAULT_RELEASE = "0.0.0"
```
Comment (jaheba) [inline src/gluonts/_version.py:28]: Should we call this `FALLBACK_RELEASE`?

Comment (arangatang) [reply src/gluonts/_version.py:28]: Sure, added it.

**Review by jaheba — APPROVED** [review]

**Pull request merged**